Jenkinsfile

```diff
@@ -178,6 +178,9 @@ node('ubuntu-zion') {
         }
         OsTools.runSafe(this, "git tag -d ${version}")
       }
+
+      //Trigger the Red Hat job and It does not care about this result
+      build(job: '/integrations/Red Hat Certified Docker Image/iq-redhat-certified', wait: false)
```

Comment:
i think it might be nice to have this wrapped in its own `stage('Run downstream')` or something like that.

i think it's fine to have to have this protected by this check for master and push_image, since nothing downstream is prepared to run for anything but master. 👍

Reply:
@jflinchbaugh stage added